static/css/less/colors.less

```diff
@@ -84,6 +86,10 @@
 @white: hsl(0, 0%, 100%);
 @gray-a19b9e: hsl(330, 3%, 62%);
 @gray-d1cfd0: hsl(330, 2%, 82%);
+@grey-e5e5e5: hsl(0, 0%, 90%);
```

Comment:
Let's use consistent spelling for the colors. The establish spelling is on the line immediately above.